lib/project.js

```diff
@@ -82,6 +90,27 @@ function Project$name() {
     return path.basename(this.rootPath);
 }
 
+function Project$license() {
```

Comment:
It's my own. It's pretty much only used here, and I didn't want to rewrite the whole file just for this change.